File: PGS/Lv.1/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
#include <algorithm>

using namespace std;

bool solution(string s)
{
	bool answer = true;

	int p = 0, y = 0;
	for (auto i : s) {
		if (i == 'P' || i == 'p') {
			p++;
		}
		if (i == 'Y' || i == 'y') {
			y++;
		}
	}

	return p == y;
}

File: PGS/Lv.1/JAVA/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        
        s = s.toLowerCase();
        int p = s.length() - s.replace("p", "").length();   // p 제거 후 두 문자열 길이의 차
        int y = s.length() - s.replace("y", "").length();   // y 제거 후 두 문자열 길이의 차

        return p == y;
    }
}